Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM rust:bullseye AS builder
```

Comment:
FYI, `bullseye` is the new Debian stable. `rust:bullseye` points to the latest version of Rust on `bullseye`.

Reply:
thanks for the info